Gui/opensim/view/src/org/opensim/view/motions/MotionDisplayer.java

```diff
@@ -147,7 +147,7 @@
     OpenSimContext dContext; 
     ArrayStr stateNames;
     private boolean renderMuscleActivations=false;
-    private double experimentalMarkerScaleFactor;
+    private double experimentalMarkerSize=1;
```

Comment:
Since the value is set in the constructor, the initialization doesn't do anything anyway, I usually initialize to non-zero value so that if it doesn't get set in constructor I can still see it (member variables initialized to 0 by default)

Reply:
@aseth1 using NaN in visualization is problematic since many other things can go wrong and lead to objects not shown, same as 0 which would happen if no initial value is specified. The value is set in the constructor so the initial value is irrelevant other than debugging but I made it into a 1000 so if future changes breaks the constructor, the object is still visible.